src/Core/Grid/Definition/GridDefinition.php

```diff
@@ -124,6 +126,21 @@ public function getColumns()
         return $this->columns;
     }
 
+    /**
+     * {@inheritdoc}
+     */
+    public function getColumn(string $id): ColumnInterface
```

Comment:
```suggestion
    public function getColumnById(string $id): ColumnInterface
```

Reply:
I thought its quite obvious :thinking: 
then should I also use same style in Column->getOption() ? (getOptionByName instead ?)

Reply:
applied getColumn --> getColumnById.
Option left as it was as it has only one identifier (whether column has also a name besides an id)